osf/models/registrations.py

```diff
@@ -647,11 +899,101 @@ def reject(self, user):
         self.approval.save()
 
     def add_status_log(self, user, action):
-        log = DraftRegistrationLog(action=action, user=user, draft=self)
+        params = {
+            'draft_registration': self._id,
+        },
+        log = DraftRegistrationLog(action=action, user=user, draft=self, params=params)
         log.save()
 
     def validate_metadata(self, *args, **kwargs):
         """
         Validates draft's metadata
         """
         return self.registration_schema.validate_metadata(*args, **kwargs)
+
+    def add_log(self, action, params, auth, save=True):
+        """
+        Tentative - probably need to combine with add_status_log
+        """
```

Comment:
Already had an `add_status_log` method.  The reason I haven't combined `add_log` with`add_status_log` yet is the additional refactoring since one uses auth and one uses user.  Verify that they should be combined (and that we should be adding all these logs to DraftRegistrations before refactoring.